cg/meta/compress/compress.py

```diff
@@ -112,7 +112,9 @@ def decompress_spring(self, sample_id: str) -> bool:
         for compression in compressions:
             if not self.crunchy_api.is_spring_decompression_possible(compression_obj=compression):
                 LOG.info(f"SPRING to FASTQ decompression not possible for {sample_id}")
-                if self.crunchy_api.is_compression_pending(compression):
+                if self.crunchy_api.is_compression_pending(
+                    compression
+                ) or self.crunchy_api.is_spring_decompression_done(compression):
```

Comment:
Nice! Do we handle the case where the compression is ongoing?

Reply:
@seallard I think "is_compression_pending" is the way we currently check that, at least if we go by how this method is set up: 
<img width="602" alt="image" src="https://github.com/Clinical-Genomics/cg/assets/40887124/b6e73f19-7c13-4dad-ba9a-d736f9352104">

Reply:
So pending means ongoing as well!